### PR TITLE
Nazjatar artifacts

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added naga artifacts.

# How to test:
Use console command "add_artifact artifact name" to add all artifacts and make sure they have icon and localization.
**List of artifact names:**
- `naga_scimitar`
- `naga_mace`
- `naga_scepter`
- `naga_trident`
- `statue_of_azshara `
- `queens_decree`
- `eternal_bouquet`
- `taxidermied_murloc`

You can check balance here: https://docs.google.com/spreadsheets/d/1xDjRHfG-ovKVcEQVfv02BsK3j0dgwS2rfH7KJ4GirA0/edit#gid=1038522838

### CODE REVIEW

**Review by arithon — APPROVED** [review]
loc LGTM

File: localisation/00_wc_artifact_localisation.csv [hunk 1]
```diff
@@ -164,6 +164,22 @@ whispers_intrigue_3;Murmurs of the Deep Below;Murmures des profondeurs;Raunen au
...
+naga_mace;Naga Mace;;Naga-Streitkolben;;;;;;;;;;;x
+naga_mace_desc;Forged in the legendary waterforges of Nazjatar.;;Geschmiedet in den legend�ren Wasserschmieden von Nazjatar.;;;;;;;;;;;x
+naga_scepter;Naga Scepter;;Naga-Szepter;;;;;;;;;;;x
+naga_scepter_desc;Made by a powerful and terrible naga sea witch.;;Hergestellt von einer m�chtigen und schrecklichen Naga-Meerhexe.;;;;;;;;;;;x
+statue_of_azshara;Statue of Azshara;;Statue von Azshara;;;;;;;;;;;x
+statue_of_azshara_desc;Small statue portraying the magnificent visage of Her Radiance Queen Azshara.;;Eine kleine Statue, die das pr�chtige Antlitz Ihrer strahlenden Majest�t K�nigin Azshara darstellt.;;;;;;;;;;;x
+queens_decree;Queens Decree;;Dekret der K�nigin;;;;;;;;;;;x
+queens_decree_desc;Numinous words of Her Radiance Queen Azshara, Queen of the Naga and Empress of Nazjatar.;;Klangvolle Worte Ihrer strahlenden Majest�t K�nigin Azshara, K�nigin der Naga und Kaiserin von Nazjatar.;;;;;;;;;;;x
+eternal_bouquet;Eternal Bouquet;;Ewiger Strau�;;;;;;;;;;;x
+eternal_bouquet_desc;Made of the most beautiful flowers from the depths of Great Sea.;;Gemacht aus den sch�nsten Blumen aus den Tiefen des Gro�en Meeres.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_wc_artifact_localisation.csv:180]: Suggesting a small fix.
```suggestion
eternal_bouquet_desc;Made of the most beautiful flowers from the depths of the Great Sea.;;Gemacht aus den sch�nsten Blumen aus den Tiefen des Gro�en Meeres.;;;;;;;;;;;x
```

**Review by LordSilvermane — APPROVED** [review]
All localization and icons are there. LGTM!

Made a small loc suggestion.

**Comment by MemeWizard** [comment]
Use `command_modifier` block for... command modifiers. They don't work otherwise.

<details>
<summary>Click to expand</summary>

image

</details>

**Comment by sejtalk** [comment]
> Use `command_modifier` block for... command modifiers. They don't work otherwise.
> 
> Click to expand

fixed

File: localisation/00_wc_artifact_localisation.csv [hunk 2]
```diff
@@ -164,6 +164,22 @@ whispers_intrigue_3;Murmurs of the Deep Below;Murmures des profondeurs;Raunen au
...
+naga_trident_desc;Forged in the legendary waterforges of Nazjatar.;;Geschmiedet in den legend�ren Wasserschmieden von Nazjatar.;;;;;;;;;;;x
+naga_scimitar;Naga Scimitar;;Naga-Krumms�bel;;;;;;;;;;;x
+naga_scimitar_desc;Forged in the legendary waterforges of Nazjatar.;;Geschmiedet in den legend�ren Wasserschmieden von Nazjatar.;;;;;;;;;;;x
+naga_mace;Naga Mace;;Naga-Streitkolben;;;;;;;;;;;x
+naga_mace_desc;Forged in the legendary waterforges of Nazjatar.;;Geschmiedet in den legend�ren Wasserschmieden von Nazjatar.;;;;;;;;;;;x
+naga_scepter;Naga Scepter;;Naga-Szepter;;;;;;;;;;;x
+naga_scepter_desc;Made by a powerful and terrible naga sea witch.;;Hergestellt von einer m�chtigen und schrecklichen Naga-Meerhexe.;;;;;;;;;;;x
+statue_of_azshara;Statue of Azshara;;Statue von Azshara;;;;;;;;;;;x
+statue_of_azshara_desc;Small statue portraying the magnificent visage of Her Radiance Queen Azshara.;;Eine kleine Statue, die das pr�chtige Antlitz Ihrer strahlenden Majest�t K�nigin Azshara darstellt.;;;;;;;;;;;x
+queens_decree;Queens Decree;;Dekret der K�nigin;;;;;;;;;;;x
```
[the remark this answers is not in the document]
Comment (arithon) [reply localisation/00_wc_artifact_localisation.csv:177]: Good that we got you :D
Yes, should be with '

**Review by MemeWizard — APPROVED** [review]

**Pull request merged**